torchx/Makefile

```diff
@@ -4,6 +4,7 @@ TORCHX_SO = $(PRIV_DIR)/torchx.so
 # If you want verbose output for debugging
 # CMAKE_BUILD_FLAGS = --verbose
 CMAKE_BUILD_DIR ?= $(MIX_APP_PATH)/cmake
+CMAKE_TOOLCHAIN_FILE ?= $(shell pwd)/toolchain.cmake
```

Comment:
This makefile is already Unix-specific, so it's probably fine to use /dev/null as the default

Reply:
Oh, that's true. Let's use /dev/null instead of adding an empty file.